examples/crowd-funding/README.md

```diff
@@ -0,0 +1,237 @@
+<!-- cargo-rdme start -->
+
+# Crowd-funding Example Application
+
+This example application implements crowd-funding campaigns using fungible tokens in
+the `fungible` application. This demonstrates how to compose applications together and
+how to instantiate applications where one chain has a special role.
+
+Once this application is built and its bytecode published on a Linera chain, the
+published bytecode can be used to create different instances, where each
+instance represents a different campaign.
+
+# How It Works
+
+The chain that created the campaign is called the "campaign chain". It is owned by the
+creator (and beneficiary) of the campaign.
+
+The goal of a crowd-funding campaign is to let people pledge any number of tokens from
+their own chain(s). If enough tokens are pledged before the campaign expires, the
+campaign is *successful* and the creator can receive all the funds. Otherwise, the
+campaign is *unsuccessful* and contributors should be refunded.
+
+# Caveat
+
+Currently, only the owner of the campaign can create blocks that contain the `Cancel`
+operation. In the future, campaign chains will not be single-owner chains and should
+instead allow contributors (users with a pledge) to cancel the campaign if
+appropriate (even without the owner's cooperation).
+
+Optionally, contributors may also be able to create a block to accept a new epoch
+(i.e. a change of validators).
+
+# Usage
+
+## Setting Up
+
+The WebAssembly binaries for the bytecode can be built and published using [steps from the
+book](https://linera-io.github.io/linera-documentation/getting_started/first_app.html),
+summarized below.
+
+First, setup a local network with two wallets, and keep it running in a separate terminal:
+
+```bash
+./scripts/run_local.sh
+```
+
+Compile the Wasm binaries for the two applications `fungible` and `crowd-funding`
+application, and publish them as an application bytecode:
```

Comment:
I would put the command used for compilation.

Reply:
It's included in the listing below:

```bash
cargo build && cd examples && cargo build --release
```